src/util/experimentPreFetch.js

```diff
@@ -18,6 +18,8 @@ let activeExperiments = [
 	// 'mg_promo',
 	'double_arrow_button',
 	'intercom_messenger',
+	//experiment setting value for recommended loans category row
+	'recommendation_channel',
```

Comment:
won't need this as it's not a global experiment. You'll want to query for the experiment setting here: /kiva/ui/src/graphql/query/lendByCategory/lendByCategory.graphql. The setting will come from settings manager. you can examples by filtering on uiexp.

Reply:
cool! thanks the reference!